### PR TITLE
feat: Add openrpc changes based on the requirements

### PR DESCRIPTION
**Pull request by satlead**

### CODE REVIEW

File: src/openrpc/device.json [hunk 1]
```diff
@@ -805,6 +1038,82 @@
...
+        "description": "The audio format supported by the platform"
+      },
+      "AudioChannels": {
+        "title": "AudioChannels",
+        "type": "string",
+        "enum": [
+          "MONO",
+          "STEREO",
+          "SURROUND",
+          "SURROUND_7_1"
```
Comment (alkalinecoffee) [inline src/openrpc/device.json:1114]: This is missing `SURROUND_5_1`

File: src/openrpc/media-info.json [hunk 2]
```diff
@@ -275,6 +273,41 @@
                     "unknown",
                     "none"
                 ]
+            },
+            "MimeType": {
+                "type": "string",
+                "description": "Enumeration of possible Mimetypes.",
+                "enum": [
```
Comment (alkalinecoffee) [inline src/openrpc/media-info.json:280]: Do methods returning MIME types need to respond with the internal name (AUDIO_AC3) or value (audio/ac3)?

File: src/openrpc/device.json [hunk 3]
```diff
@@ -544,6 +544,239 @@
...
+          "result": {
+            "name": "Default Result",
+            "value": true
+          }
+        }
+      ]
+    },
+    {
+      "name": "videoFormatPossible",
+      "summary": "Get the possible video formats",
```
Comment (alkalinecoffee) [inline src/openrpc/device.json:607]: This method is a check, it does not return any video formats

File: src/openrpc/device.json [hunk 4]
```diff
@@ -544,6 +544,239 @@
...
+          "result": {
+            "name": "Default Result",
+            "value": true
+          }
+        }
+      ]
+    },
+    {
+      "name": "audioFormatPossible",
+      "summary": "Check the possible video formats",
```
Comment (alkalinecoffee) [inline src/openrpc/device.json:723]: Summary is for video but this is an audio method

File: src/openrpc/display.json [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+                    "name": "capabilities",
+                    "x-uses": [
+                        "xrn:firebolt:capability:display:info"
+                    ]
+                }
+            ],
+            "params": [],
+            "result": {
+                "name": "height",
+                "summary": "Height of the display device in centimeters",
```
Comment (alkalinecoffee) [inline src/openrpc/display.json:83]: The name and summary here are incorrect

File: src/openrpc/display.json [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+                    "name": "capabilities",
+                    "x-uses": [
+                        "xrn:firebolt:capability:display:info"
+                    ]
+                }
+            ],
+            "params": [],
+            "result": {
+                "name": "resolution",
+                "summary": "Height of the display device in centimeters",
```
Comment (alkalinecoffee) [inline src/openrpc/display.json:113]: The summary is incorrect; this returns a resolution

**Comment by satlead** [comment]
PR will be merged to the feature branch and work will progress in the main feature branch

**Pull request merged**